catalog.bom

```diff
@@ -85,8 +147,35 @@ brooklyn.catalog:
           is.root.node: true
           launch.latch: $brooklyn:component("my-hyperledger-membersrvc-node").attributeWhenReady("service.isUp")
 
+  - id: hyperledger-fabric-cluster-single-location
+    item:
+      name: Hyperledger Fabric Cluster (Single-location)
+      type: org.apache.brooklyn.entity.stock.BasicApplication
+
+      brooklyn.children:
+      - type: org.apache.brooklyn.entity.group.DynamicRegionsFabric
```

Comment:
I agree in theory, but [https://github.com/cloudsoft/brooklyn-hyperledger/blob/single-and-multi-location/catalog.bom#L369] expects a Brooklyn fabric and I'd prefer not to make two separate entities here.